C7/MainMenu.cs

```diff
@@ -77,14 +83,14 @@ private void AddButton(string label, int verticalPosition, string actionName)
 		TextureButton newButton = new TextureButton();
 		newButton.TextureNormal = InactiveButton;
 		newButton.TextureHover = HoverButton;
-		newButton.SetPosition(new Vector2(835, verticalPosition));
+		newButton.SetPosition(new Vector2(40, verticalPosition));
```

Comment:
I moved these to the left side from the right (which was only chosen because it matched Civ III Vanilla).  This has the advantage of not appearing to jump around when you resize the screen.  It also looks better with Babylon's background (the clouds on the right make it hard to find a color that works well for the font on all buttons).

Long-term, we can anchor it better if we want it on the right for a future background.

Also long-term, the buttons/labels and their spacing don't scale with the screen size, whereas the background (including "C7 Babylon") does.  I don't think it's worth delaying Babylon over figuring out how to get these main menu labels scaling amazingly.